.storybook/main.js

```diff
@@ -7,8 +7,14 @@ const SOURCE_DIR = path.join(__dirname, '../src');
 const SOURCE_DOCS_DIR = path.join(SOURCE_DIR, '_docs');
 const SOURCE_COMPONENTS_DIR = path.join(SOURCE_DIR, 'components');
 
+console.log(process.env);
+
 module.exports = {
-  stories: ['../src/**/*.stories.@(jsx|mdx)'],
+  stories: [
+    process.env.CHROMATIC === 'true'
+      ? '../src/**/*.chromatic-stories.@(jsx|mdx)'
```

Comment:
I am thinking about changing the name of chromatic stories from `chromatic-stories` to `chromatic.stories.@(jsx|mdx)`

The only challenge will be to filter out `chromatic.stories` from normal storybook run. Fortunately stories property in main.js config also takes async function so we can dynamically collect all needed stories and filter them out in js function.

reference: https://storybook.js.org/docs/react/configure/overview#configure-story-loading

Reply:
Updated. Now we can use `.chromatic.stories` and `.stories` extensions